tests/x11/vnc_two_passwords.pm

```diff
@@ -87,7 +85,7 @@ sub run {
     send_key "ctrl-c";
     assert_script_run "sed -i '\$d' $theme";
     select_console "x11";
-    x11_start_program "rt";
+    x11_start_program('rt', target_match => 'generic-desktop');
```

Comment:
Change in behavior again. We should verify changes, if needle will work.

Reply:
Should not be a change in behaviour. The post_run_hook of every x11 test looks for "generic-desktop" anyway. I just additionally check inside x11_start_program so that we can exit early to save time.

Reply:
Agreed.